src/main/java/com/travelagency/model/Discount.java

```diff
@@ -0,0 +1,43 @@
+package com.travelagency.model;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "discount")
+public class Discount {
+
+    @Id
+    @GeneratedValue
+    @Column(name = "id")
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "trip_id")
+    private Trip trip;
+
+    @Column(name = "percentage", length = 10)
+    @NotNull
+    private int percentage;
+
+    public Discount(Trip trip, @NotNull int percentage) {
+        this.trip = trip;
+        this.percentage = percentage;
+    }
+
+    public Trip getTrip() {
+        return trip;
+    }
+
+    public void setTrip(Trip trip) {
```

Comment:
Heeft een discount een trip? Alleen een trip heeft een discount toch?

Reply:
Je hebt gelijk, het relatie is inderdaad van een kant.